### PR TITLE
Shorthand properties examples

### PR DESCRIPTION
**Pull request by bhoule**
The shorthand properties example output did not match the actual output

| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | 
| Major: Breaking Change?  | 
| Minor: New Feature?      | 
| Deprecations?            | 
| Spec Compliancy?         | 
| Tests Added/Pass?        | 
| Fixed Tickets            |
| License                  | MIT
| Doc PR                   | yes
| Dependency Changes       |

### CODE REVIEW

File: packages/babel-plugin-transform-es2015-shorthand-properties/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ var cat = {
 
 ```js
 var cat = {
-  getName: function () {
+  getName: function getName() {
```
Comment (existentialism) [inline packages/babel-plugin-transform-es2015-shorthand-properties/README.md]: Actually, the current example is correct... the REPL currently only lets you select presets, and not individual plugins... the function is getting its name from `transform-es2015-function-name`.

(p.s. we'd love for someone to help us augment the REPL to fix this!)

Comment (xtuc) [inline packages/babel-plugin-transform-es2015-shorthand-properties/README.md]: When you run Try on the [plugin's page on the website](https://babeljs.io/docs/plugins/transform-es2015-shorthand-properties/). The preset es2015 is enable by default.

Comment (existentialism) [inline packages/babel-plugin-transform-es2015-shorthand-properties/README.md]: I know but it seems weird to say that the output for this specific transform is what the preset outputs and not the plugin?

Comment (bhoule) [inline packages/babel-plugin-transform-es2015-shorthand-properties/README.md]: I agree. I'll limit the change to just the missing space on line 16.

**Comment by existentialism** [comment]
Thanks @bhoule!

File: packages/babel-plugin-transform-es2015-shorthand-properties/README.md (hunk 1 above)
Comment (xtuc) [inline packages/babel-plugin-transform-es2015-shorthand-properties/README.md]: OK for me

**Pull request merged**